### PR TITLE
Update workers changelog

### PR DESCRIPTION
**Pull request by dom96**
This is from the 2024.8.1700 release.

CC @Frederik-Baetens

### CODE REVIEW

**Review by irvinebroque — APPROVED** [review]

File: src/content/changelogs/workers.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ productLink: "/workers/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+- publish_date: '2024-09-07'
+  description: |-
+    -  Introduced a new overloaded error message for Durable Objects: "Durable Object is overloaded. Too many requests for the same object within a 10 second window." This error message does not replace other types of overload messages that you may encounter for your Durable Object, and is only returned at more extreme levels of overload.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml:10]: Durable Objects has its own changelog here:

https://developers.cloudflare.com/durable-objects/platform/changelog/

Should it be there? @vy-ton

Comment (vy-ton) [reply src/content/changelogs/workers.yaml:10]: Yes, should go in DO changelog - https://developers.cloudflare.com/durable-objects/platform/changelog/#2024-06-24

**Comment by Oxyjun** [comment]
Hi, I've created a separate PR to close this one out. 

https://github.com/cloudflare/cloudflare-docs/pull/17271

**Comment by dom96** [comment]
Thank you @Oxyjun!

**Pull request closed without merging**